### ISSUE TITLE
Hook for dynamic config

### ISSUE THREAD

**Issue by devinivy** [BigRoomStudios/strangeluv#23]
We can use something like trumpet to inject config into the page.  This would allow us to avoid compiling the application with the environment baked-in, for greater portability and to mitigate risk of non-matching config when deploying alongside other hapi plugins.  For example, if an API and front-end both rely on `API_PREFIX` then currently the value set at front-end compile-time needs to match the value set for API deploy-time.

**Comment by jdonisvitch** [comment]
Is this still relevant?

**Comment by wswoodruff** [comment]
Yeah I think so! This looks like a good idea to me could help avoid config inconsistencies like he mentioned

**Comment by jdonisvitch** [comment]
Maybe this can be something we take on in the first quarter of upcoming dev time?
